draft-ietf-dnsop-svcb-https.md

```diff
@@ -424,7 +426,7 @@ request.  This limit MUST NOT be zero, i.e. implementations MUST be able to
 follow at least one AliasMode record.  The exact value of this limit
 is left to implementations.
 
-For compatibility and performance, zone owners SHOULD NOT configure their zones
+For compatibility and performance, zone owners should not configure their zones
```

Comment:
I feel like lower-casing the keyword doesn't really address the issue here.  This is a legitimate recommendation.  Either we keep the SHOULD NOT -- but when might someone want to do this? -- or we rephrase this as a statement of fact:  "A zone that requires following multiple AliasMode records could encounter compatibility and performance issues."

Reply:
Done